backend/src/main/kotlin/app/ehrenamtskarte/backend/mail/Mailer.kt

```diff
@@ -310,7 +318,7 @@ object Mailer {
                 )
             }
             p { +"Dieser Link ist 24 Stunden gültig." }
-            finalInformationParagraph(projectConfig)
+            finalInformationParagraph()
```

Comment:
For this mail and for the mail below, we have to use a more generic way, as these are also necessary for Nürnberg & Koblenz.

Reply:
I'm pretty sure we don't send any emails for the KoblenzPass. The entitled person doesn't have to provide an email address and there is also none contained in the provided data.
For Nürnberg we don't manage the applications at all, so there shouldn't be any email communication as well.

Reply:
ah, indeed! 🤦‍♀️
thanks @michael-markl 
@connium is it fine if we keep it as Ehrenamtskarte-Bayern-Verwaltung for Bayern?

Reply:
we could also introduce another string variable for mail templates for each project

Reply:
@connium any suggestions for signature for Koblenz and Nürnberg?
Ihr Team von SocialPass Nürnberg/Koblenz ?

Reply:
Ihr Team des Nürnberg-Passes/KoblenzPasses